lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -118,8 +118,12 @@
                                 {trigger: true}
                             );
                         })
-                        .fail(function() {
-                            var message = gettext('An error occurred. Please try again.');
+                        .fail(function(data) {
+                            var response = JSON.parse(data.responseText);
+                            var message = gettext("An error occurred. Please try again.")
+                            if ('error_message' in response){
```

Comment:
We don't always put a user_message... sometimes there is just a developer_message (this is allowed by our REST API guidelines). It would be good to make the conditional also check that there is a user_message.